examples/ota-requestor-app/esp32/README.md

```diff
@@ -31,67 +31,41 @@ idf.py -p <OTARequestorSerialPort> flash monitor
 -   Commissioning the OTA Requestor
 
 ```
-./out/debug/chip-tool pairing ble-wifi 12346 <ssid> <passphrase> 0 20202021 3840
+./out/debug/chip-tool pairing ble-wifi 12346 <ssid> <passphrase> 20202021 3840
 ```
 
 ## Query for an OTA Image
 
-After commissioning is successful, query for OTA image. Head over to ESP32
-console and fire the following command. This command creates a CASE session with
-OTA Provider and start the OTA image transfer using BDX protocol.
+After commissioning is successful, announce OTA provider's presence using
+chip-tool. On receiving this command OTA requestor will query for OTA image.
 
 ```
-esp32> QueryImage <OTAProviderIpAddress> <OTAProviderNodeId>
-```
-
-OTA image transfer takes some time and once done it prints following log
-
-```
-I (322620) OTARequesterImpl: Transfer complete!
+./out/debug/chip-tool otasoftwareupdaterequestor announce-ota-provider 12345 0 0 12346 0
 ```
 
 ## Apply update request
 
 Once transfer is complete OTA Requestor should take permission from the OTA
-Provider for applying the OTA image.
+Provider for applying the OTA image. Use the following command from OTA
+requestor prompt
 
 ```
-esp32> ApplyUpdateRequest <OTAProviderIpAddress> <OTAProviderNodeId>
+esp32> ApplyUpdateRequest
 ```
 
-After this step device should reboot and start running hello world example.
+Then reboot the device manually to boot from upgraded OTA image.
```

Comment:
The README changes look fine. Although, I am not quite convinced with this :-| Why do we have to manually reboot after OTA is done. Is this only for now and will be update/fixed later on?

Reply:
May be we can reboot automatically after `delayedActionTime`?